src/app/Scenes/PartnerOffer/PartnerOfferContainer.tsx

```diff
@@ -0,0 +1,51 @@
+import { Flex } from "@artsy/palette-mobile"
+import { usePartnerOfferCheckoutMutation$data } from "__generated__/usePartnerOfferCheckoutMutation.graphql"
+import { LoadingSpinner } from "app/Components/Modals/LoadingModal"
+import { usePartnerOfferMutation } from "app/Scenes/PartnerOffer/mutations/usePartnerOfferCheckoutMutation"
+import { goBack, navigate } from "app/system/navigation/navigate"
+import { useEffect } from "react"
+
+export const PartnerOfferContainer: React.FC<{ partnerOfferID: string }> = ({ partnerOfferID }) => {
+  const handleRedirect = (response: usePartnerOfferCheckoutMutation$data) => {
+    const orderOrError = response.commerceCreatePartnerOfferOrder?.orderOrError
+
+    if (orderOrError?.error) {
+      const { code, data } = orderOrError.error
```

Comment:
too late for this but `error_code` would have been a nice name for this. 
```suggestion
      const { code: errorCode, data: maybeArtwork } = orderOrError.error
```

Reply:
maybe `data:artwork` could also be better here if we're sure there will be an artwork returned - since the offered artwork might get deleted or unpublished